connectors/sink-tencentcloud-scf/internal/server.go

```diff
@@ -33,19 +32,26 @@ import (
 )
 
 const (
-	name = "Tencent Cloud COS Source"
+	name = "Tencent SCF Sink"
 )
 
 var (
 	functionNamePrefix = "vanus-cos-source-function"
 )
 
+var _ config.SinkConfigAccessor = &Config{}
+
 type Config struct {
+	config.SinkConfig
```

Comment:
```suggestion
	cdkgo.SinkConfig
```
cdkgo has some alias, you can reference it